code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -706,7 +718,7 @@
 /singleton/reagent/sodiumchloride/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
 	overdose(M, alien, removed, holder)
 
-/singleton/reagent/sodiumchloride/overdose(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
+/singleton/reagent/sodiumchloride/overdose(mob/living/carbon/M, alien, removed, datum/reagents/holder)
```

Comment:
If you randomly change one of those you have to change all of them in the same file.

With that said, we don't have a style guide requiring the use one or the other.